library/std/src/sys/sync/thread_parking/futex.rs

```diff
@@ -1,15 +1,18 @@
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Please use forbid to deny this once and for all.
```suggestion
#![forbid(unsafe_op_in_unsafe_fn)]
```